src/pe_reports/__init__.py

```diff
@@ -20,19 +20,25 @@
 
 # Stakeholder views
 from pe_reports.home.views import home_blueprint
+from pe_reports.report_gen.views import report_gen_blueprint
 from pe_reports.stakeholder.views import stakeholder_blueprint
 
 from ._version import __version__  # noqa: F401
 
 params = config()
+"""Test to see if port is empty."""
+if params["host"] == "":
```

Comment:
@DJensen94 , 
Did you mean to put 'port' here?

Reply:
addressed in this commit: [97918e7](https://github.com/cisagov/pe-reports/pull/282/commits/97918e79991c15de3a80e61b117fa988ffce2a79)